pom.xml

```diff
@@ -9,125 +9,145 @@
     <version>1.0-SNAPSHOT</version>
     <packaging>war</packaging>
 
+    <properties>
+        <spring.core>4.3.8.RELEASE</spring.core>
```

Comment:
usually it written like "spring.core.version", and as for spring you can use "spring.version" because spring usually has one version across all products